store/src/java/com/zimbra/cs/service/mail/SendShareNotification.java

```diff
@@ -80,6 +91,12 @@ public class SendShareNotification extends MailDocumentHandler {
     }
 
     private static final String REVOKE = "revoke";
+    private static final String EDIT = "edit";
+    private static final String CREATE = "edit";
+
+    public static final String ZIMBRA_SOAP_USER_AGENT = "ZimbraSync";
+    public static final String ZIMBRA_SOAP_VERSION = "1.0";
```

Comment:
aren't these already defined, almost all Soap apis will be using this right?

Reply:
I couldn't find out where exactly these are defined , check in SoapHTTPTransport class . couldn't find any similar constant . if you any reference it would be helpful

Reply:
the method using this was removed , hence this removed this constants as well